File: 코딩 기초 트레이닝/2의 영역-p181894/2의 영역.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181894

def solution(arr):
    answer = []
    flag = False

    if arr.count(2) == 0:
        return [-1]
    elif arr.count(2) == 1:
        return [2]
    else:
        first = arr.index(2)
        arr.reverse()
        last = arr.index(2)
        arr.reverse()
        answer = arr[first:-last]

    return arr if len(answer) == 0 else answer

File: 코딩 기초 트레이닝/2의 영역-p181894/2의 영역.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181894

function solution(arr) {
    var answer = [];

    let first = arr.indexOf(2);
    let last = arr.lastIndexOf(2);

    if(first === -1) {
        return [-1];
    } else if(first === last) {
        return [2];
    } else {
        return arr.slice(first, last+1)
    }

    return answer;
}
